app_dart/lib/src/model/github/checks.dart

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:github/github.dart' hide CheckSuite;
+import 'package:github/hooks.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'checks.g.dart';
+
+@JsonSerializable(fieldRename: FieldRename.snake)
```

Comment:
Nit: provide links to github checks api in comments.

Reply:
done